README.pypi.md

```diff
@@ -31,6 +31,15 @@ import sys, subprocess
 subprocess.call([sys.executable, "-m", "ziglang"])
 ```
 
+Usage through uv
+----------------
+
+Zig-pypi also exposes a Python tool that wraps the zig compiler so that build systems like [uv](https://docs.astral.sh/uv) can easily run the zig compiler:
+
+```shell
+uvx --from ziglang python-zig
+```
+
```

Comment:
```suggestion
Binary wrapper
--------------

The [ziglang][pypi] Python package installs a binary wrapper for the Zig compiler under the name `python-zig`; the name is different to avoid conflicts with any system-wide or user-wide `zig` binaries that may be already installed.


Using with `uv`
---------------

The Zig compiler distributed in this Python package can be launched by [uv](https://docs.astral.sh/uv) without installation:

```shell
uvx --from ziglang python-zig
```

```